cpp/benchmarks/CMakeLists.txt

```diff
@@ -256,6 +256,11 @@ ConfigureNVBench(
   PARQUET_READER_NVBENCH io/parquet/parquet_reader_input.cpp io/parquet/parquet_reader_options.cpp
 )
 
+# ##################################################################################################
+# * parquet multithread reader benchmark
+# ----------------------------------------------------------------------
+ConfigureNVBench(PARQUET_MULTITHREAD_READER_NVBENCH io/parquet/parquet_reader_multithread.cpp)
```

Comment:
I think it's a little easier to just have them separate.  Partially because this benchmark suite is itself pretty large, and partially because the thing we're using it to measure is fairly different (inter-kernel performance related issues rather than intra-kernel)

Reply:
I don't mind this being separate. I would not want to run these every time I check PQ performance.